api/src/integrations/core/datasource.js

```diff
@@ -25,10 +25,12 @@ class CoreDataSource {
    * Creates an instance of CoreDataSource.
    *
    * @param {*} taskAuditRecord audit record hook for this import instance
+   * @param {*} auth_payload information about the user account that started this update.
    * @memberof CoreDataSource
    */
-  constructor(taskAuditRecord) {
+  constructor(taskAuditRecord, auth_payload) {
```

Comment:
On the other objects the auth_payload is the param. Might be nice to make them align.

Reply:
I wanted to limit the amount of adjustments I made - cleanup ticket perhaps?